pkg/applier/applier.go

```diff
@@ -770,7 +770,7 @@ func (a *supervisor) handleDisabledObjects(ctx context.Context, rg *live.Invento
 	for _, obj := range objs {
 		id := core.IDOf(obj)
 		err := a.abandonObject(ctx, obj)
-		handleMetrics(ctx, "unmanage", err, obj.GetObjectKind().GroupVersionKind())
+		handleMetrics(ctx, "unmanage", err, obj.GetObjectKind().GroupVersionKind().Kind)
```

Comment:
Is this always Unstructured? If not, the GVK might not be set. Should we check?

Reply:
I traced the root where `objs` was initialized. It is unstructured, but with GVK populated: https://github.com/GoogleContainerTools/kpt-config-sync/blob/main/pkg/kinds/convert.go#L35. We should be good here.  We can also use `id.Kind` for simplicity.